wrappers/services.go

```diff
@@ -62,6 +62,9 @@ func serviceStopTimeout(app *snap.AppInfo) time.Duration {
 	return time.Duration(tout)
 }
 
+// TODO: this should not accept AddSnapServicesOptions, it should use some other
+// subset of options, specifically it should not accept Preseeding as an option
+// here
```

Comment:
I don't think we particularly worry not to send Preseeding to this one. I think passing both kind of options directly would work, it's just a bit annoying to adapt the tests.  Anyway can be done later in a follow up.